OrbitGl/TimerTrack.cpp

```diff
@@ -107,11 +107,22 @@ void TimerTrack::UpdatePrimitives(uint64_t min_tick, uint64_t max_tick,
 
         bool is_visible_width = normalized_length * canvas->GetWidth() > 1;
         bool is_selected = &text_box == GOrbitApp->selected_text_box();
+        bool need_highlight = GOrbitApp->selected_text_box() &&
+                              timer_info.thread_id() != SamplingProfiler::kAllThreadsFakeTid &&
+                              timer_info.thread_id() == GOrbitApp->selected_thread_id() &&
```

Comment:
This looks like it's only highlighting timers on the selected thread, shouldn't we highlight on every thread?

Reply:
Thanks for the suggestion!